plugin.json

```diff
@@ -37,4 +45,4 @@
     "header": "",
     "footer": ""
   }
-}
+}
```

Comment:
Nit pick: Why remove the newline?

Reply:
It is my auto-formatter

Reply:
Would you mind manually reverting your change?

Reply:
Not at all :-)